rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -181,24 +182,14 @@ impl FirmwareProcessor {
     ///
     /// * `Manifest` - Caliptra Image Bundle Manifest
     #[cfg_attr(not(feature = "no-cfi"), cfi_impl_fn)]
-    fn load_manifest(txn: &mut MailboxRecvTxn) -> CaliptraResult<ImageManifest> {
-        let slice = unsafe {
-            let ptr = MAN1_ORG as *mut u32;
-            core::slice::from_raw_parts_mut(ptr, core::mem::size_of::<ImageManifest>() / 4)
-        };
-
-        txn.copy_request(slice.as_bytes_mut())?;
-
-        let opt = ImageManifest::read_from(slice.as_bytes());
-        let result = opt.is_some();
-        if cfi_launder(result) {
```

Comment:
NOTE: This CFI check doesn't really make sense; opt will only be None when the slice is smaller than slice.len()*4, which is impossible.

Perhaps this code was expecting read_from() to some sort of validation?